tankersdk_identity/__init__.py

```diff
@@ -6,12 +6,25 @@
 import tankersdk_identity.crypto
 
 
+APP_SECRET_SIZE = 64
+APP_PUBLIC_KEY_SIZE = 32
+AUTHOR_SIZE = 32
+APP_CREATION_NATURE = 1
+
+
 def _hash_user_id(app_id, user_id):
     user_id_buff = user_id.encode()
     to_hash = user_id_buff + app_id
     return tankersdk_identity.crypto.generichash(to_hash, size=BLOCK_HASH_SIZE)
 
 
+def _generate_app_id(app_secret):
+    public_key = app_secret[APP_SECRET_SIZE - APP_PUBLIC_KEY_SIZE:APP_SECRET_SIZE]
+    # A bit convoluted, to make it work with python 2.7
```

Comment:
Not sure we still need to support it...

Reply:
Continuing maintaining Python2.7 for sdk-identity is not a huge burden for us, so let's keep it working for now

Reply:
yeah, it's really no big deal